code/game/turfs/turf.dm

```diff
@@ -257,6 +253,10 @@
 		else
 			lighting_clear_overlays()
 
+	W.levelupdate()
+	W.CalculateAdjacentTurfs()
```

Comment:
Hmmm; wonder how TG is handling this. air_update_turf calls CalculateAdjacnetTurfs, so its' not a big deal---but ideally, something else isn't properly being called somewhere...